### PR TITLE
English blog post on how to contribute

### PR DESCRIPTION
**Pull request by yabellini**
English version on the blog post on how to contribute to rOpenSci.  I will translate to Spanish when we agree on the English version.

### CODE REVIEW

File: content/blog/2022-09-08-contributing/index.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
+---
+title: How to get involved with rOpenSci 
+author: Yanina Bellini Saibene
+date: '2022-09-07T00:00:01-07:00'
+slug: contributing-ropensci
+tags:
+  - community
+feature: true  
+# description: 
+tweet: "Are you curious about @rOpenSci? Do you wonder where I get started participating? 🤔 Glad you asked! A summary on how to get involved with us!"
```
Comment (maelle) [inline content/blog/2022-09-08-contributing/index.md:10]:
```suggestion
tweet: "Are you curious about @rOpenSci? Wondering how to start participating? 🤔 Glad you asked! A summary on how to get involved with us!" 
```

File: content/blog/2022-09-08-contributing/index.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
+---
+title: How to get involved with rOpenSci
```
Comment (maelle) [inline content/blog/2022-09-08-contributing/index.md:2]: I wonder whether we should rather use "participate" (not sure).

File: content/blog/2022-09-08-contributing/index.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+
+## Where Do I Get Started?
+
+Glad you asked! rOpenSci fosters a culture that values open and reproducible research using shared data and reusable software. We are dedicated to providing a welcoming and supportive environment for all people. By participating in this community, participants accept to [abide by our Code of Conduct](/code-of-conduct/). Whether you have a little or a lot of time, there are [many ways to contribute and engage with us](https://contributing.ropensci.org/).
+
+If you're ready for the big dive, check out our [Contributing Guide](https://contributing.ropensci.org/) which covers all the ways you can get more involved with rOpenSci and how you may benefit. 
+
+Otherwise, see below for a quick overview of the resources available.
+
```
Comment (maelle) [inline content/blog/2022-09-08-contributing/index.md:27]:
```suggestion
```

File: content/blog/2022-09-08-contributing/index.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+
+## Asking for Help
+
+Do you want to know How to use an rOpenSci Package? or are you looking information on How to develop an R package?, there are several way to ask for help:
+
```
Comment (maelle) [inline content/blog/2022-09-08-contributing/index.md:32]:
```suggestion
```

File: content/blog/2022-09-08-contributing/index.md [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+
+* [Discussion forum](https://discuss.ropensci.org/): public forum on Discourse for use cases, Q & A about rOpenSci packages, exploration of ideas and best practices
```
Comment (maelle) [inline content/blog/2022-09-08-contributing/index.md:34]: Maybe link to https://ropensci.org/blog/2022/01/11/ropensci-forum/

File: content/blog/2022-09-08-contributing/index.md [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+
+* [Packages Documentation](https://docs.ropensci.org/): This server hosts pkgdown documentation for all rOpenSci packages.
+
+* [Use Cases](https://ropensci.org/usecases/): check how our R packages or resources have been used.
+
+* [Technical Notes](https://ropensci.org/technotes/): browse our technical notes to see packages updates, how-to and examples.
```
Comment (maelle) [inline content/blog/2022-09-08-contributing/index.md:40]: Maybe rather link to the blog in general?

File: content/blog/2022-09-08-contributing/index.md [hunk 7]
```diff
@@ -0,0 +1,91 @@
...
+## Where Do I Get Started?
+
+Glad you asked! rOpenSci fosters a culture that values open and reproducible research using shared data and reusable software. We are dedicated to providing a welcoming and supportive environment for all people. By participating in this community, participants accept to [abide by our Code of Conduct](/code-of-conduct/). Whether you have a little or a lot of time, there are [many ways to contribute and engage with us](https://contributing.ropensci.org/).
+
+If you're ready for the big dive, check out our [Contributing Guide](https://contributing.ropensci.org/) which covers all the ways you can get more involved with rOpenSci and how you may benefit. 
+
+Otherwise, see below for a quick overview of the resources available.
+
+
+## Asking for Help
```
Comment (maelle) [inline content/blog/2022-09-08-contributing/index.md:29]:
```suggestion
## Finding information & asking for Help
```

File: content/blog/2022-09-08-contributing/index.md [hunk 8]
```diff
@@ -0,0 +1,91 @@
...
+* [rOpenSci Statistical Software Peer Review](https://stats-devguide.ropensci.org/): this book provides guidelines for authors on how to develop statistical software, and for editors and reviewers on our processes for peer review of statistical software.
+
+* _Slack_: A semi-open space for communications among people who contribute to rOpenSci. Our Slack is [open to anyone who contributes](https://ropensci.org/contact/) to an rOpenSci project or participates in our peer-review process as an author or reviewer.
+
+
+## Chat with Other Community Members
+
+If you want to chat with other members of the community, in addition to the [Discussion forum](https://discuss.ropensci.org/) and _Slack_, we have meeting places such as
+ 
+* [Community Calls](https://discuss.ropensci.org/): join to our quarterly Community Calls, these are free and open for anyone to attend and provide an opportunity to connect with rOpenSci community members around the world
```
Comment (maelle) [inline content/blog/2022-09-08-contributing/index.md:53]:
```suggestion
* [Community Calls](/commcalls): join to our quarterly Community Calls, these are free and open for anyone to attend and provide an opportunity to connect with rOpenSci community members around the world.
```

File: content/blog/2022-09-08-contributing/index.md [hunk 9]
```diff
@@ -0,0 +1,91 @@
...
+
+* [Remote Co-Working Sessions](https://ropensci.org/blog/2021/08/17/coworking-sessions/): monthly social coworking + office hours sessions via Zoom
```
Comment (maelle) [inline content/blog/2022-09-08-contributing/index.md:55]:
```suggestion
* [Remote Co-Working Sessions](/blog/2021/08/17/coworking-sessions/): monthly social coworking + office hours sessions via Zoom
```

File: content/blog/2022-09-08-contributing/index.md [hunk 10]
```diff
@@ -0,0 +1,91 @@
...
+* [Newsletter](https://ropensci.org/news/): every month received in your inbox R package news, blog posts, tips and tricks, events and use cases. Subscribe via RSS (XML), JSON feed, or email.
+
+* _Social Media_: Follow our organization’s activities on [Twitter](https://twitter.com/rOpenSci), and [Vimeo](https://vimeo.com/ropensci). Also follow our [roknowtifier bot](https://twitter.com/roknowtifier) that tweets new rOpenSci packages or new versions of packages
+
+
+## Use and contribute to our infrastructure
+
+* [Packages](https://ropensci.org/packages/): our packages are carefully vetted, staff- and community-contributed R software tools that in particular lower barriers to working with local and remote scientific data sources.
+
+* [The R-universe](https://ropensci.org/r-universe/): improving publication and discovery of research software in R.
```
Comment (maelle) [inline content/blog/2022-09-08-contributing/index.md:72]:
```suggestion
* [The R-universe](/r-universe/): improving publication and discovery of research software in R.
```

File: content/blog/2022-09-08-contributing/index.md [hunk 11]
```diff
@@ -0,0 +1,91 @@
...
+We have several channels for our information to reach you directly. Worried about too many communications? Don’t be! Only subscribe to the communications you want, and you can unsubscribe at any time.
+
+* [Newsletter](https://ropensci.org/news/): every month received in your inbox R package news, blog posts, tips and tricks, events and use cases. Subscribe via RSS (XML), JSON feed, or email.
+
+* _Social Media_: Follow our organization’s activities on [Twitter](https://twitter.com/rOpenSci), and [Vimeo](https://vimeo.com/ropensci). Also follow our [roknowtifier bot](https://twitter.com/roknowtifier) that tweets new rOpenSci packages or new versions of packages
+
+
+## Use and contribute to our infrastructure
+
+* [Packages](https://ropensci.org/packages/): our packages are carefully vetted, staff- and community-contributed R software tools that in particular lower barriers to working with local and remote scientific data sources.
```
Comment (maelle) [inline content/blog/2022-09-08-contributing/index.md:70]:
```suggestion
* [Packages](/packages/): our packages are carefully vetted, staff- and community-contributed R software tools that in particular lower barriers to working with local and remote scientific data sources.
```

File: content/blog/2022-09-08-contributing/index.md [hunk 12]
```diff
@@ -0,0 +1,91 @@
...
+
+* [The R-universe](https://ropensci.org/r-universe/): improving publication and discovery of research software in R.
+
+* [Package Review](https://ropensci.org/software-review/): packages contributed by the community undergo a transparent, constructive, non-adversarial and open review process.
```
Comment (maelle) [inline content/blog/2022-09-08-contributing/index.md:74]:
```suggestion
* [Software Peer-Review](/software-review/): packages contributed by the community undergo a transparent, constructive, non-adversarial and open review process.
```

File: content/blog/2022-09-08-contributing/index.md [hunk 13]
```diff
@@ -0,0 +1,91 @@
...
+
+* [Packages Statistical Review](https://ropensci.org/stat-software-review/): we have expanded our peer review system to include packages that implement statistical algorithms.
```
Comment (maelle) [inline content/blog/2022-09-08-contributing/index.md:76]:
```suggestion
* [Statistical Software Peer-Review](/stat-software-review/): we have expanded our peer review system to include packages that implement statistical algorithms.
```

File: content/blog/2022-09-08-contributing/index.md [hunk 14]
```diff
@@ -0,0 +1,91 @@
...
+
+* [Multilingual Publishing](https://ropensci.org/blog/2021/12/20/inclusive-leadership-program/): we are expanding documentation for our review process beyond the English language.
```
Comment (maelle) [inline content/blog/2022-09-08-contributing/index.md:78]:
```suggestion
* [Multilingual Publishing](/blog/2021/12/20/inclusive-leadership-program/): we are expanding documentation for our review process beyond the English language.
```

File: content/blog/2022-09-08-contributing/index.md [hunk 15]
```diff
@@ -0,0 +1,91 @@
...
+
+* [Champions Program](https://ropensci.org/blog/2021/12/20/inclusive-leadership-program/): support community champions from historically excluded groups to become contributors to rOpenSci.
```
Comment (maelle) [inline content/blog/2022-09-08-contributing/index.md:80]:
```suggestion
* [Champions Program](/blog/2021/12/20/inclusive-leadership-program/): support community champions from historically excluded groups to become contributors to rOpenSci.
```

File: content/blog/2022-09-08-contributing/index.md [hunk 16]
```diff
@@ -0,0 +1,91 @@
...
+
+* [Package Citations](https://ropensci.org/citations/): We manually curate a dataset of all citations of our packages. If you use one of our packages, please cite it (learn here how).
```
Comment (maelle) [inline content/blog/2022-09-08-contributing/index.md:82]: I would not mention this as it is currently not curated.

Comment (maelle) [reply content/blog/2022-09-08-contributing/index.md:82]: But at the same time it is always good to encourage citations, so linking to https://ropensci.org/blog/2021/11/16/how-to-cite-r-and-r-packages/ would be nice.

File: content/blog/2022-09-08-contributing/index.md [hunk 17]
```diff
@@ -0,0 +1,91 @@
...
+
+* [Help Wanted](https://ropensci.org/help-wanted/): if you are ready to contribute, here is a list of issues where your expertise is needed (we have issues for different levels of expertise).
```
Comment (maelle) [inline content/blog/2022-09-08-contributing/index.md:84]:
```suggestion
* [Help Wanted](/help-wanted/): if you are ready to contribute, here is a list of issues where your expertise is needed (we have issues for different levels of expertise).
```

File: content/blog/2022-09-08-contributing/index.md [hunk 18]
```diff
@@ -0,0 +1,91 @@
...
+
+## Additional Resources
+
+* [Talks and Publications](https://ropensci.org/talks-papers/): talks by the rOpenSci team or community members representing us, in English, French, or Spanish.
```
Comment (maelle) [inline content/blog/2022-09-08-contributing/index.md:88]:
```suggestion
* [Talks and Publications](/talks-papers/): talks by the rOpenSci team or community members representing us, in English, French, or Spanish.
```

File: content/blog/2022-09-08-contributing/index.md [hunk 19]
```diff
@@ -0,0 +1,91 @@
...
+
+* [Blog](https://ropensci.org/blog/): articles written by our community. Our [guide explains how to contribute a blog post](https://blogguide.ropensci.org/).
```
Comment (maelle) [inline content/blog/2022-09-08-contributing/index.md:90]:
```suggestion
* [Blog](/blog/): articles written by our community. Our [guide explains how to contribute a blog post](https://blogguide.ropensci.org/).
```

**Review by maelle — CHANGES_REQUESTED** [review]
Thank you, this will be a super useful post!

File: content/blog/2022-09-08-contributing/index.md [hunk 20]
```diff
@@ -0,0 +1,89 @@
...
+
+* [Software Peer-Review](/software-review/): packages contributed by the community undergo a transparent, constructive, non-adversarial and open review process.
+
+* [Statistical Software Peer-Review](/stat-software-review/): we have expanded our peer review system to include packages that implement statistical algorithms.
+
+* [Multilingual Publishing](/blog/2021/12/20/inclusive-leadership-program/): we are expanding documentation for our review process beyond the English language.
+
+* [Champions Program](/blog/2021/12/20/inclusive-leadership-program/): support community champions from historically excluded groups to become contributors to rOpenSci.
+
+* [Package Citations](https://ropensci.org/citations/): We manually curate a dataset of all citations of our packages. If you use one of our packages, please cite it (learn here how).
```
Comment (yabellini) [inline content/blog/2022-09-08-contributing/index.md:80]:
```suggestion
* [Package Citations](/citations/): If you use one of our packages, please cite it! [You can learn here how to cite R and R Packages](https://ropensci.org/blog/2021/11/16/how-to-cite-r-and-r-packages/).
```

File: content/blog/2022-09-08-contributing/index.md [hunk 21]
```diff
@@ -0,0 +1,89 @@
...
+
+If you're ready for the big dive, check out our [Contributing Guide](https://contributing.ropensci.org/) which covers all the ways you can get more involved with rOpenSci and how you may benefit. 
+
+Otherwise, see below for a quick overview of the resources available.
+
+## Finding information & asking for Help
+
+Do you want to know How to use an rOpenSci Package? or are you looking information on How to develop an R package?, there are several way to ask for help:
+
+* [Discussion forum](https://discuss.ropensci.org/): public forum on Discourse for use cases, Q & A about rOpenSci packages, exploration of ideas and best practices
```
Comment (yabellini) [inline content/blog/2022-09-08-contributing/index.md:32]:
```suggestion
* [Discussion forum](https://discuss.ropensci.org/): public forum on Discourse for use cases, Q & A about rOpenSci packages, exploration of ideas, and best practices. Learn [how to gather on the rOpenSci forum](/blog/2022/01/11/ropensci-forum/)
```

File: content/blog/2022-09-08-contributing/index.md [hunk 22]
```diff
@@ -0,0 +1,89 @@
+---
+title: How to get involved with rOpenSci
```
Comment (yabellini) [inline content/blog/2022-09-08-contributing/index.md:2]:
```suggestion
title: How to participate with rOpenSci 
```

File: content/blog/2022-09-08-contributing/index.md [hunk 23]
```diff
@@ -0,0 +1,89 @@
...
+
+Do you want to know How to use an rOpenSci Package? or are you looking information on How to develop an R package?, there are several way to ask for help:
+
+* [Discussion forum](https://discuss.ropensci.org/): public forum on Discourse for use cases, Q & A about rOpenSci packages, exploration of ideas, and best practices. Learn [how to gather on the rOpenSci forum](/blog/2022/01/11/ropensci-forum/)
+
+* [Packages Documentation](https://docs.ropensci.org/): This server hosts pkgdown documentation for all rOpenSci packages.
+
+* [Use Cases](https://ropensci.org/usecases/): check how our R packages or resources have been used.
+
+* [Technical Notes](https://ropensci.org/technotes/): browse our technical notes to see packages updates, how-to and examples.
```
Comment (yabellini) [inline content/blog/2022-09-08-contributing/index.md:38]:
```suggestion
* [Blog](/blog/): browse our technical notes to see packages updates, how-to and examples. You can find articles written by our community members. If you feel inspired, our [guide explains how to contribute a blog post](https://blogguide.ropensci.org/).
```

File: content/blog/2022-09-08-contributing/index.md [hunk 24]
```diff
@@ -0,0 +1,89 @@
...
+* [Packages Documentation](https://docs.ropensci.org/): This server hosts pkgdown documentation for all rOpenSci packages.
+
+* [Use Cases](https://ropensci.org/usecases/): check how our R packages or resources have been used.
+
+* [Blog](/blog/): browse our technical notes to see packages updates, how-to and examples. You can find articles written by our community members. If you feel inspired, our [guide explains how to contribute a blog post](https://blogguide.ropensci.org/).
+
+* [rOpenSci Packages: Development, Maintenance, and Peer Review](https://devguide.ropensci.org/): this book contains our guidelines for creating and testing R packages and our process for peer review.
+
+* [rOpenSci Statistical Software Peer Review](https://stats-devguide.ropensci.org/): this book provides guidelines for authors on how to develop statistical software, and for editors and reviewers on our processes for peer review of statistical software.
+
```
Comment (yabellini) [inline content/blog/2022-09-08-contributing/index.md:43]: * [Talks and Publications](/talks-papers/): talks by the rOpenSci team or community members representing us, in English, French, or Spanish.

File: content/blog/2022-09-08-contributing/index.md [hunk 25]
```diff
@@ -0,0 +1,89 @@
...
+* [Multilingual Publishing](/blog/2021/12/20/inclusive-leadership-program/): we are expanding documentation for our review process beyond the English language.
+
+* [Champions Program](/blog/2021/12/20/inclusive-leadership-program/): support community champions from historically excluded groups to become contributors to rOpenSci.
+
+* [Package Citations](/citations/): If you use one of our packages, please cite it! [You can learn here how to cite R and R Packages](https://ropensci.org/blog/2021/11/16/how-to-cite-r-and-r-packages/).
+
+* [Help Wanted](/help-wanted/): if you are ready to contribute, here is a list of issues where your expertise is needed (we have issues for different levels of expertise).
+
+## Additional Resources
```
Comment (yabellini) [inline content/blog/2022-09-08-contributing/index.md:84]:
```suggestion

```

File: content/blog/2022-09-08-contributing/index.md [hunk 26]
```diff
@@ -0,0 +1,88 @@
...
+* [Multilingual Publishing](/blog/2021/12/20/inclusive-leadership-program/): we are expanding documentation for our review process beyond the English language.
+
+* [Champions Program](/blog/2021/12/20/inclusive-leadership-program/): support community champions from historically excluded groups to become contributors to rOpenSci.
+
+* [Package Citations](/citations/): If you use one of our packages, please cite it! [You can learn here how to cite R and R Packages](https://ropensci.org/blog/2021/11/16/how-to-cite-r-and-r-packages/).
+
+* [Help Wanted](/help-wanted/): if you are ready to contribute, here is a list of issues where your expertise is needed (we have issues for different levels of expertise).
+
+
+* [Talks and Publications](/talks-papers/): talks by the rOpenSci team or community members representing us, in English, French, or Spanish.
```
Comment (yabellini) [inline content/blog/2022-09-08-contributing/index.md:85]:
```suggestion

```

File: content/blog/2022-09-08-contributing/index.md [hunk 27]
```diff
@@ -0,0 +1,87 @@
...
+
+* [Champions Program](/blog/2021/12/20/inclusive-leadership-program/): support community champions from historically excluded groups to become contributors to rOpenSci.
+
+* [Package Citations](/citations/): If you use one of our packages, please cite it! [You can learn here how to cite R and R Packages](https://ropensci.org/blog/2021/11/16/how-to-cite-r-and-r-packages/).
+
+* [Help Wanted](/help-wanted/): if you are ready to contribute, here is a list of issues where your expertise is needed (we have issues for different levels of expertise).
+
+
+
+* [Blog](/blog/): articles written by our community. Our [guide explains how to contribute a blog post](https://blogguide.ropensci.org/).
```
Comment (yabellini) [inline content/blog/2022-09-08-contributing/index.md:86]:
```suggestion

```

File: content/blog/2022-09-08-contributing/index.md [hunk 28]
```diff
@@ -0,0 +1,87 @@
...
+* [Packages Documentation](https://docs.ropensci.org/): This server hosts pkgdown documentation for all rOpenSci packages.
+
+* [Use Cases](https://ropensci.org/usecases/): check how our R packages or resources have been used.
+
+* [Blog](/blog/): browse our technical notes to see packages updates, how-to and examples. You can find articles written by our community members. If you feel inspired, our [guide explains how to contribute a blog post](https://blogguide.ropensci.org/).
+
+* [rOpenSci Packages: Development, Maintenance, and Peer Review](https://devguide.ropensci.org/): this book contains our guidelines for creating and testing R packages and our process for peer review.
+
+* [rOpenSci Statistical Software Peer Review](https://stats-devguide.ropensci.org/): this book provides guidelines for authors on how to develop statistical software, and for editors and reviewers on our processes for peer review of statistical software.
+
```
Comment (yabellini) [inline content/blog/2022-09-08-contributing/index.md:43]:
```suggestion
* [Talks and Publications](https://github.com/talks-papers/): talks by the rOpenSci team or community members representing us, in English, French, or Spanish.
```

**Review by steffilazerte — COMMENTED** [review]
Hi @yabellini 

Looks lovely! I've been opinionated about some things, feel free to ignore (I don't mind :grin:). There are a couple of stylistic things (title should be title case, section headings in sentence case) that I've tweaked but generally not much.

Thanks!

* [X] post follows Content Guidelines
* [ ] post follows Style Guide
* [ ] title is in Title Case
* [ ] publication date is ok
* [X] alternative text of images is informative
* [X] Twitter metadata looks ok (paste post preview link in [Twitter card validator](https://cards-dev.twitter.com/validator); might have to click twice on "Preview card")
* [X] author metadata is provided with correct folder name
* [X] html not included in pull request of Rmd post
* [X] I ran `roblog::ro_lint_md()` on [index.md](http://index.md/)
* [X] I ran `roblog::ro_check_urls()` on [index.md](http://index.md/)
* [X] I ran a spell-check on [index.md](http://index.md/)
* [X] YAML subject tags are ok ("tech notes" for tech notes; "community" for non-staff non-editor)

File: content/blog/2022-09-08-contributing/index.md [hunk 29]
```diff
@@ -0,0 +1,86 @@
...
+* [Multilingual Publishing](/blog/2021/12/20/inclusive-leadership-program/): we are expanding documentation for our review process beyond the English language.
+
+* [Champions Program](/blog/2021/12/20/inclusive-leadership-program/): support community champions from historically excluded groups to become contributors to rOpenSci.
+
+* [Package Citations](/citations/): If you use one of our packages, please cite it! [You can learn here how to cite R and R Packages](https://ropensci.org/blog/2021/11/16/how-to-cite-r-and-r-packages/).
+
+* [Help Wanted](/help-wanted/): if you are ready to contribute, here is a list of issues where your expertise is needed (we have issues for different levels of expertise).
+
+
```
[the remark this answers is not in the document]
Comment (yabellini) [reply content/blog/2022-09-08-contributing/index.md:84]: Yes, agree we need a close for the blog post.

File: content/blog/2022-09-08-contributing/index.md [hunk 30]
```diff
@@ -0,0 +1,89 @@
...
+* [Packages Documentation](https://docs.ropensci.org/): This site hosts pkgdown documentation for all rOpenSci packages.
+
+* [Use Cases](/usecases/): check how our R packages or resources have been used.
+
+* [Blog](/blog/): browse our technical notes to see packages updates, how-to and examples. You can find articles written by our community members. If you feel inspired, our [guide explains how to contribute a blog post](https://blogguide.ropensci.org/).
+
+* [rOpenSci Packages: Development, Maintenance, and Peer Review](https://devguide.ropensci.org/): this book contains our guidelines for creating and testing R packages and our process for peer review.
+
+* [rOpenSci Statistical Software Peer Review](https://stats-devguide.ropensci.org/): this book provides guidelines for authors on how to develop statistical software, and for editors and reviewers on our processes for peer review of statistical software.
+* [Talks and Publications](https://github.com/talks-papers/): talks by the rOpenSci team or community members representing us, in English, French, or Spanish.
```
Comment (steffilazerte) [inline content/blog/2022-09-08-contributing/index.md:46]: Just realized that this link doesn't exist... not sure what it should be though!

**Comment by steffilazerte** [comment]
One more comment about a broken link, sorry I missed it before!

File: content/blog/2022-09-08-contributing/index.md (hunk 30 above)
Comment (yabellini) [inline content/blog/2022-09-08-contributing/index.md:46]:
```suggestion
* [Talks and Publications](https://ropensci.org/talks-papers/): talks by the rOpenSci team or community members representing us, in English, French, or Spanish.
```

File: content/blog/2022-09-08-contributing/index.md [hunk 31]
```diff
@@ -0,0 +1,89 @@
...
+
+* [Multilingual Publishing](/blog/2021/12/20/inclusive-leadership-program/): we are expanding documentation for our review process beyond the English language.
+
+* [Champions Program](/blog/2021/12/20/inclusive-leadership-program/): support community champions from historically excluded groups to become contributors to rOpenSci.
+
+* [Package Citations](/citations/): If you use one of our packages, please cite it! [You can learn here how to cite R and R Packages](/blog/2021/11/16/how-to-cite-r-and-r-packages/).
+
+* [Help Wanted](/help-wanted/): if you are ready to contribute, here is a list of issues where your expertise is needed (we have issues for different levels of expertise).
+
+
```
Comment (yabellini) [inline content/blog/2022-09-08-contributing/index.md:87]: ## Ready to contribute!

We are happy you made it this far.  We hope you find some way to participate in the options.  We would be glad to learn more if you have ideas about other ways to collaborate with us.

We hope to see you in some of our spaces.

File: content/blog/2022-09-08-contributing/index.md [hunk 32]
```diff
@@ -0,0 +1,89 @@
...
```
Comment (yabellini) [inline content/blog/2022-09-08-contributing/index.md:87]:
```suggestion
## Ready to contribute!

We are happy you made it this far.  We hope you find some way to participate in the options.  We would be glad to learn more if you have ideas about other ways to collaborate with us.

We hope to see you in some of our spaces.
```

**Comment by yabellini** [comment]
Thanks so much Steffi, I think I should change the date and then we are ok?

File: content/blog/2022-09-08-contributing/index.md [hunk 33]
```diff
@@ -0,0 +1,93 @@
...
+* [Champions Program](/blog/2021/12/20/inclusive-leadership-program/): support community champions from historically excluded groups to become contributors to rOpenSci.
+
+* [Package Citations](/citations/): If you use one of our packages, please cite it! [You can learn here how to cite R and R Packages](/blog/2021/11/16/how-to-cite-r-and-r-packages/).
+
+* [Help Wanted](/help-wanted/): if you are ready to contribute, here is a list of issues where your expertise is needed (we have issues for different levels of expertise).
+
+## Ready to contribute!
+
+We are happy you made it this far.  We hope you find some way to participate in the options.  We would be glad to learn more if you have ideas about other ways to collaborate with us.
```
Comment (steffilazerte) [inline content/blog/2022-09-08-contributing/index.md:89]:
```suggestion
We are happy you made it this far.  We hope you find some way to participate among these options.  We would be glad to learn more if you have ideas about other ways to collaborate with us!
```

File: content/blog/2022-09-08-contributing/index.md [hunk 34]
```diff
@@ -0,0 +1,93 @@
...
+
+* [Software Peer-Review](/software-review/): packages contributed by the community undergo a transparent, constructive, non-adversarial and open review process.
+
+* [Statistical Software Peer-Review](/stat-software-review/): we have expanded our peer review system to include packages that implement statistical algorithms.
+
+* [Multilingual Publishing](/blog/2021/12/20/inclusive-leadership-program/): we are expanding documentation for our review process beyond the English language.
+
+* [Champions Program](/blog/2021/12/20/inclusive-leadership-program/): support community champions from historically excluded groups to become contributors to rOpenSci.
+
+* [Package Citations](/citations/): If you use one of our packages, please cite it! [You can learn here how to cite R and R Packages](/blog/2021/11/16/how-to-cite-r-and-r-packages/).
```
Comment (steffilazerte) [inline content/blog/2022-09-08-contributing/index.md:83]:
```suggestion
* [Package Citations](/citations/): if you use one of our packages, please cite it! [You can learn here how to cite R and R Packages](/blog/2021/11/16/how-to-cite-r-and-r-packages/).
```

File: content/blog/2022-09-08-contributing/index.md [hunk 35]
```diff
@@ -0,0 +1,93 @@
...
+
+* [Events](/events/): our staff and collaborators participate in different events and conferences.
+
+## Receive communications
+
+We have several channels for our information to reach you directly. Worried about too many communications? Don’t be! Only subscribe to the communications you want, and you can unsubscribe at any time.
+
+* [Newsletter](/news/): every month receive in your inbox R package news, blog posts, tips and tricks, events and use cases. Subscribe via RSS (XML), JSON feed, or email.
+
+* _Social Media_: Follow our organization’s activities on [Twitter](https://twitter.com/rOpenSci), and [Vimeo](https://vimeo.com/ropensci). Also follow our [roknowtifier bot](https://twitter.com/roknowtifier) that tweets new rOpenSci packages or new versions of packages
```
Comment (steffilazerte) [inline content/blog/2022-09-08-contributing/index.md:66]:
```suggestion
* _Social Media_: follow our organization’s activities on [Twitter](https://twitter.com/rOpenSci), and [Vimeo](https://vimeo.com/ropensci). Also follow our [roknowtifier bot](https://twitter.com/roknowtifier) that tweets new rOpenSci packages or new versions of packages
```

File: content/blog/2022-09-08-contributing/index.md [hunk 36]
```diff
@@ -0,0 +1,93 @@
...
+
+Otherwise, see below for a quick overview of the resources available.
+
+## Finding information & asking for help
+
+Do you want to know how to use an rOpenSci package? or are you looking information on how to develop an R package? There are several way to ask for help:
+
+* [Discussion forum](https://discuss.ropensci.org/): public forum on Discourse for use cases, Q & A about rOpenSci packages, exploration of ideas, and best practices. Learn how to[Gather on the rOpenSci Forum](/blog/2022/01/11/ropensci-forum/)
+
+* [Packages Documentation](https://docs.ropensci.org/): This site hosts pkgdown documentation for all rOpenSci packages.
```
Comment (steffilazerte) [inline content/blog/2022-09-08-contributing/index.md:37]:
```suggestion
* [Packages Documentation](https://docs.ropensci.org/): this site hosts pkgdown documentation for all rOpenSci packages.
```

**Comment by steffilazerte** [comment]
Yup! Just a couple of last nit picky formating and one wording suggestion, other than that, pick at date (don't forget to change both the folder date and the YAML date), and we're good to go!

**Comment by yabellini** [comment]
The translation is ready too, so if you think is ok we can publish both languages.

File: content/blog/2022-09-13-contributing/index.md [hunk 37]
```diff
@@ -0,0 +1,93 @@
...
+  - community
+  - help
+  - beginner
+  - welcome
+feature: true  
+# description: 
+tweet: "Are you curious about @rOpenSci? Wondering how to start participating? 🤔 Glad you asked! Here's a summary on how to get involved with us!" 
+---
+
+[Entrada en español](/blog/2022/09/13/contributing-es/)
```
Comment (steffilazerte) [inline content/blog/2022-09-13-contributing/index.md:16]:
```suggestion
[Entrada en español](/blog/2022/09/13/contributing-ropensci-es/)
```

File: content/blog/2022-09-13-contributing-es/index.md [hunk 38]
```diff
@@ -0,0 +1,96 @@
...
+  - beginner
+  - welcome
+  - español
+  - spanish
+feature: true  
+# description: 
+tweet: "¿Tienes curiosidad por @rOpenSci? Te preguntas cómo empezar a participar? 🤔 ¡Nos alegramos! Aquí tienes un resumen de cómo participar con nosotros" 
+---
+
+[Post in English](/blog/2022/09/13/contributing-es/)
```
Comment (steffilazerte) [inline content/blog/2022-09-13-contributing-es/index.md:18]:
```suggestion
[Post in English](/blog/2022/09/13/contributing-ropensci/)
```

**Comment by steffilazerte** [comment]
The English / español links need to be adjusted (see my suggestions) but other than that I'd publish!

File: content/blog/2022-09-13-contributing-es/index.md [hunk 39]
```diff
@@ -0,0 +1,96 @@
+---
+title: Cómo participar en rOpenSci
```
Comment (steffilazerte) [inline content/blog/2022-09-13-contributing-es/index.md:2]:
```suggestion
title: Cómo Participar en rOpenSci
```

Comment (yabellini) [reply content/blog/2022-09-13-contributing-es/index.md:2]: In Spanish we don't capitalize words in titles. If we need it now I'm ok but we should change that rule for spanish

Comment (steffilazerte) [reply content/blog/2022-09-13-contributing-es/index.md:2]: Oh that makes sense, I was wondering. I think it makes sense to have different style guides for different languages.

**Comment by yabellini** [comment]
Thanks for spot the wrong links! I commit those changes. For the title we can change for this one but un Spanish we don't capitalize words in titles. Only the first one (and mames).

**Comment by steffilazerte** [comment]
I wouldn't change the title then, keep it how it should be in Spanish. Our current style guide makes sense for English but not for Spanish!

**Comment by yabellini** [comment]
Then we are ready to go!

**Pull request merged**